server/config/routes/api/v3.rb

```diff
@@ -0,0 +1,36 @@
+Fusor::Engine.routes.draw do
```

Comment:
We have a v21 and a v3?  v21 has changed significantly, we might want to consider putting everything in a v3 since old code that worked on v21 won't work on v21 anymore.